### PR TITLE
[SLICING] PBI-23: See Lesson Material

### PR DESCRIPTION
**Pull request by fxdros**
Screen Shot 2022-04-17 at 21 23 19

### CODE REVIEW

**Comment by FXDROS** [comment]
Screen Shot 2022-04-17 at 21 23 19

File: src/resources/app/pages/course/CourseMaterial.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  </CreationLayout>
+</template>
+
+<script>
+import AppLayout from "../../components/Layout/AppLayout";
+import CreationLayout from "../../components/Layout/CreationLayout";
+
+export default {
+  name: "CourseMaterial",
+  components: {CreationLayout},
```
Comment (AldiNFitrah) [inline src/resources/app/pages/course/CourseMaterial.vue:25]: nit: spaces between curly braces

File: tests/unit/CourseMaterial.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+
+import {createLocalVue, mount} from "@vue/test-utils";
```
Comment (AldiNFitrah) [inline tests/unit/CourseMaterial.spec.js:4]: same

**Review by AldiNFitrah — APPROVED** [review]

**Review by IlmaMannix — APPROVED** [review]
Good job, just reminder to handle "only teachers who can edit and delete the material"

**Comment by AldiNFitrah** [comment]
> Good job, just reminder to handle "only teachers who can edit and delete the material"

I guess you meant that only CREATOR can edit and delete the material

**Pull request merged**